INSTALL.md

```diff
@@ -183,6 +183,15 @@ please open an issue.
 emerge --onlydeps x11-terms/alacritty
 ```
 
+#### GNU Guix
+
+Run following command to enter a shell with all development dependencies
+on [GNU Guix](https://guix.gnu.org/).
```

Comment:
```suggestion
The following command can be used to get a shell with all development
dependencies on [GNU Guix](https://guix.gnu.org/).
```

Let's just copy our wording from the above `NixOS/Nixpkgs` section.

Reply:
Done